tests/contrib/anthropic/test_anthropic.py

```diff
@@ -0,0 +1,139 @@
+            model="claude-3-opus-20240229",
+            max_tokens=15,
+            messages=[
+                {
+                    "role": "user",
+                    "content": [
+                        {"type": "text", "text": "Hello, I am looking for information about some books!"},
+                        {"type": "text", "text": "Can you explain what Descartes meant by 'I think, therefore I am'?"},
+                    ],
+                }
+            ],
+        )
+
+
+@pytest.mark.snapshot()
+def test_anthropic_llm_sync_multiple_prompts_with_chat_history(anthropic, request_vcr):
+    llm = anthropic.Anthropic()
+    with request_vcr.use_cassette("anthropic_completion_sync_multi_prompt_with_chat_history.yaml"):
+        llm.messages.create(
+            model="claude-3-opus-20240229",
+            max_tokens=30,
+            messages=[
+                {
+                    "role": "user",
+                    "content": [
+                        {"type": "text", "text": "Hello, Start all responses with your name Claude."},
+                        {"type": "text", "text": "End all responses with [COPY, CLAUDE OVER AND OUT!]"},
+                    ],
+                },
+                {"role": "assistant", "content": "Claude: Sure! [COPY, CLAUDE OVER AND OUT!]"},
+                {
+                    "role": "user",
+                    "content": [
+                        {
+                            "type": "text",
+                            "text": "Add the time and date to the beginning of your response after your name.",
+                        },
+                        {"type": "text", "text": "Explain string theory succinctly to a complete noob."},
+                    ],
+                },
+            ],
+        )
+
+
+@pytest.mark.snapshot(ignores=["meta.error.stack"])
+def test_anthropic_llm_error(anthropic, request_vcr):
+    llm = anthropic.Anthropic()
+    invalid_error = anthropic.BadRequestError
+    with pytest.raises(invalid_error):
+        with request_vcr.use_cassette("anthropic_completion_error.yaml"):
+            llm.messages.create(model="claude-3-opus-20240229", max_tokens=15, messages=["Invalid content"])
+
+
+@pytest.mark.snapshot()
+def test_anthropic_llm_sync_stream(anthropic, request_vcr):
+    llm = anthropic.Anthropic()
+    with request_vcr.use_cassette("anthropic_completion_sync_stream.yaml"):
+        stream = llm.messages.create(
+            model="claude-3-opus-20240229",
+            max_tokens=15,
+            messages=[
+                {
+                    "role": "user",
+                    "content": [
+                        {
+                            "type": "text",
+                            "text": "Can you explain what Descartes meant by 'I think, therefore I am'?",
+                        }
+                    ],
+                },
+            ],
+            stream=True,
+        )
+        for chunk in stream:
+            print(chunk.type)
```

Comment:
```suggestion
            pass
```
We don't want to print the test output, just passing iterating over the stream is fine